homeassistant/components/google_travel_time/config_flow.py

```diff
@@ -30,16 +35,89 @@
     DEPARTURE_TIME,
     DOMAIN,
     TIME_TYPES,
+    TRAFFIC_MODELS,
     TRANSIT_PREFS,
-    TRANSPORT_TYPE,
-    TRAVEL_MODE,
-    TRAVEL_MODEL,
+    TRANSPORT_TYPES,
+    TRAVEL_MODES,
     UNITS,
     UNITS_IMPERIAL,
     UNITS_METRIC,
 )
 from .helpers import InvalidApiKeyException, UnknownException, validate_config_entry
 
+NONE_SENTINEL = "none"
```

Comment:
```suggestion
```

Similar to https://github.com/home-assistant/core/pull/101278
frontend PR will be merged after `2023.10.0` is released